Adjusted FAQ page code to improve page efficiency and code conventions.

kea-roy opened the pull request:
### Summary <!-- Required -->

Adjusted FAQ page code to improve page efficiency and code conventions.
1. Changed name of constant "dark-red" to "red-7"
2. Moved useStyles constant outside of collapsiblequestion component and adjusted code to support mobile styling based on className instead of boolean variable.
3. Moved useStyles constant outside of faq page component and adjusted code to support mobile styling based on className instead of boolean variable.

ggsawatyanon reviewed (changes requested):
Good job on changing the color name. For the isMobile conditional styling, using a ternary operator and the `style` prop would help reduce repetitiveness in code. Here is an example from ReviewModal:
![image](https://github.com/cornell-dti/cu-apts/assets/125820523/8ffdc31a-a004-40d7-8879-a3f9a2547878)
![image](https://github.com/cornell-dti/cu-apts/assets/125820523/46bacfaa-2a0e-4b82-8707-182ca941652f)

kea-roy commented:
## Commit Changes
1. Undid all changes in the previous commit
2. Changed name of constant "dark-red" to "red-7"
3. Moved useStyles constant outside of collapsiblequestion component and adjusted code to support mobile styling based on className instead of boolean variable by storing desktop styling in useStyles and mobile styling in the component's style using ternary operators.
4. Moved useStyles constant outside of faq page component and adjusted code to support mobile styling based on className instead of boolean variable by storing desktop styling in useStyles and mobile styling in the component's style using ternary operators.

ggsawatyanon reviewed (approved):
Thanks again for changing everything as requested, this looks great and demonstrates good coding practice.
